zerver/tests/test_openapi.py

```diff
@@ -169,6 +169,8 @@ class OpenAPIArgumentsTest(ZulipTestCase):
         '/users/me/presence',
         '/users/me',
         '/bot_storage',
+        '/users/me/api_keys',
+        '/users/me/api_keys/{api_key_id}',
```

Comment:
I added these endpoints to undocumented list in open api test to pass the open api tests which were failing as suggested by @Hypro999 . The `documentation_pending` method was not working for url endpoints @timabbott